doc/advanced-topics.rst

```diff
@@ -95,6 +95,8 @@ set of predicates:
   it is linked as part of a driver or meant to add a ``-ppx`` argument
   to the compiler, choose the former behavior
 
+.. _advanced-cross-compilation:
```

Comment:
I don't mind adding these for explicitness, but are you sure this is required? I thought sphinx lets you refer to headers without the explicit references.